dlgr/griduniverse/experiment.py

```diff
@@ -274,19 +281,18 @@ def __init__(self, **kwargs):
         self.alternate_consumption_donation = kwargs.get(
             'alternate_consumption_donation', False)
 
-        # Food
-        self.num_food = kwargs.get('num_food', 8)
-        self.respawn_food = kwargs.get('respawn_food', True)
-        self.food_visible = kwargs.get('food_visible', True)
-        self.food_reward = kwargs.get('food_reward', 1)
-        self.food_pg_multiplier = kwargs.get('food_pg_multiplier', 1)
-        self.food_growth_rate = kwargs.get('food_growth_rate', 1.00)
-        self.food_maturation_speed = kwargs.get('food_maturation_speed', 1)
-        self.food_maturation_threshold = kwargs.get(
-            'food_maturation_threshold', 0.0)
+        # Objects
+        self.object_count = kwargs.get('object_count', 8)
```

Comment:
These shouldn't be coming in via `__init__` but being read from `self.object_config` since they are per object configuration. Generally speaking, I think we don't need to set them here since they will differ per object and need to be looked up per object when doing spawning, etc.